bleachbit/Cleaner.py

```diff
@@ -497,7 +501,8 @@ def __init__(self):
             # the uninstallers for software updates.
             self.add_option('updates', _('Update uninstallers'), _(
                 'Delete uninstallers for Microsoft updates including hotfixes, service packs, and Internet Explorer updates'))
-
+        elif 'posix' == os.name and sys.platform.startswith('darwin'):
```

Comment:
it have update the recycle bin location for mac and it does work fine.as
you can see in attached pic.


On Tue, Mar 15, 2016 at 12:22 AM, Andrew Ziem <notifications@github.com>
wrote:

> In bleachbit/Cleaner.py
> <https://github.com/az0/bleachbit/pull/129#discussion_r56113654>:
>
> > @@ -497,7 +501,8 @@ def __init__(self):
> >              # the uninstallers for software updates.
> >              self.add_option('updates', _('Update uninstallers'), _(
> >                  'Delete uninstallers for Microsoft updates including hotfixes, service packs, and Internet Explorer updates'))
> > -
> > +        elif 'posix' == os.name and sys.platform.startswith('darwin'):
>
> While this adds the option (the checkbox), I do not see a corresponding
> code to actually clean the recycle bin
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/az0/bleachbit/pull/129/files#r56113654
>